### PR TITLE
Origin/ft template rendering

### PR DESCRIPTION
**Pull request by tcarreira**
This is functional by now, so I think this could be merged

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,110 +1,111 @@
...
 	})
+	defer s.Shutdown()
+
+	// Create internal redirect server to render static files
+	http.Handle("/rooms/", handleRooms(http.FileServer(http.Dir("html_template")), s))
+
+	// Register /events endpoint
+	http.Handle("/events/", s)
+
+	// go func() {
```
Comment (gsilvapt) [inline main.go:85]: Since we are not using these two go routines, do you mind removing these?

File: main.go [hunk 2]
```diff
@@ -1,110 +1,111 @@
...
+	// 		i++
+	// 		stringJSON := `{"title": "Title for Room 2", "next": "Next Presentation With a Relative Big Text Title, so I Can Test This big Text @ 16:00", "time": "` + strconv.Itoa(i) + `"}`
+	// 		s.SendMessage("/events/channel-2", sse.SimpleMessage(stringJSON))
+	// 		time.Sleep(5 * time.Second)
+	// 	}
+	// }()
 
-	// Routing handler
-	http.Handle("/", http.FileServer(http.Dir("static_files")))
+	fmt.Println("============= Instructions ==============")
```
Comment (gsilvapt) [inline main.go:103]: And what about moving this to the README file? Or do you prefer having an actual CLI displaying this information?

**Review by gsilvapt — CHANGES_REQUESTED** [review]

**Comment by gsilvapt** [comment]
One last thing: Binary files should not be committed to the git repository. Can you remove it from git, please?

**Comment by tcarreira** [comment]
You're right. I'm gonna update this at the other PR (I just figured now I duplicated it)
gonna close this.

**Comment by gsilvapt** [comment]
> You're right. I'm gonna update this at the other PR (I just figured now I duplicated it)
> gonna close this.

I also did not realize this earlier. Yes, lets keep it clean. Remember though, I created the other PR so you have to accept the changes to merge it. I cannot do that for you :smile: But I will change these rules because organizers should be able to allow and merge these requests.

**Pull request closed without merging**